contracts/DegenLockToken.sol

```diff
@@ -34,12 +34,19 @@ contract DegenLockToken is ERC20, Ownable, ReentrancyGuard {
     /**
      * @dev Lock duration in seconds, period starts after the deposit timestamp
      */
-    uint256 public lockDuration = 90 days;
+    /**  If the lockDuration and minDepositAmount are only meant to be updated via the provided functions, consider changing their visibility to private and provide public getter functions. */
+    uint256 private lockDuration = 90 days;
```

Comment:
Public variables automatically generate a getter function, which can be used by external contracts and users to access the variable's value.